### PR TITLE
update pre-requsites to include checking docker is installed

### PR DESCRIPTION
**Pull request by pavanstarmanwar**
# Changes
- [x] Add a line to check Docker is running

```
$ kind create cluster
ERROR: failed to create cluster: failed to list nodes: command "docker ps -a --filter label=io.x-k8s.kind.cluster=kind --format '{{.Names}}'" failed with error: exit status 1
Command Output: Cannot connect to the Docker daemon at unix:///Users/USER/.docker/run/docker.sock. Is the docker daemon running?
Ms-MacBook-Pro:~$ docker ps
Cannot connect to the Docker daemon at unix:///Users/USER/.docker/run/docker.sock. Is the docker daemon running?
```
After starting docker

```
Ms-MacBook-Pro:~$ kind create cluster
Creating cluster "kind" ...
 ✓ Ensuring node image (kindest/node:v1.25.3) 🖼 
 ✓ Preparing nodes 📦  
 ✓ Writing configuration 📜 
 ✓ Starting control-plane 🕹️ 
 ✓ Installing CNI 🔌 
 ✓ Installing StorageClass 💾 
Set kubectl context to "kind-kind"
You can now use your cluster with:

kubectl cluster-info --context kind-kind

Have a question, bug, or feature request? Let us know! https://kind.sigs.k8s.io/#community 🙂
```

# Submitter Checklist

As the author of this PR, please check off the items in this checklist:

- [x] Has [Docs](https://github.com/tektoncd/community/blob/main/standards.md#docs) included if any changes are user facing
- [ ] Has [Tests](https://github.com/tektoncd/community/blob/main/standards.md#tests) included if any functionality added or changed
- [x] Follows the [commit message standard](https://github.com/tektoncd/community/blob/main/standards.md#commits)
- [x] Meets the [Tekton contributor standards](https://github.com/tektoncd/community/blob/main/standards.md) (including
  functionality, content, code)
- [x] Has a kind label. You can add one by adding a comment on this PR that contains `/kind <type>`. Valid types are bug, cleanup, design, documentation, feature, flake, misc, question, tep
- [ ] Release notes block below has been updated with any user facing changes (API changes, bug fixes, changes requiring upgrade notices or deprecation warnings)
- [ ] Release notes contains the string "action required" if the change requires additional action from users switching to the new release

# Release Notes

```release-note
NONE
```

### CODE REVIEW

**Review by jerop — COMMENTED** [review]
@pavanstarmanwar welcome to tekton!
/meow
/kind documentation
/ok-to-test

File: docs/install.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ This guide explains how to install Tekton Pipelines.
 ## Prerequisites
 
 -   A [Kubernetes cluster][k8s] running version 1.23 or later.
+-   If you are running on MAC, make sure Docker is running
```
Comment (pritidesai) [inline docs/install.md:25]: suggest replacing `MAC` with `macOS`

**Comment by pritidesai** [comment]
thank you @pavanstarmanwar for your contribution! Welcome to the tekton community 🎉 

Please submit the checklist in the PR description. The additional check looks good to me.

I would suggest (a followup PR) to include an additional section `troubleshooting` in the end of the `install.md` to include what you have in the PR description i.e. before/after. Thanks!

**Review by jerop — APPROVED** [review]

**Comment by pritidesai** [comment]
thank you @pavanstarmanwar 👍 

/lgtm

File: docs/install.md [hunk 2]
```diff
@@ -22,6 +22,7 @@ This guide explains how to install Tekton Pipelines.
 ## Prerequisites
 
 -   A [Kubernetes cluster][k8s] running version 1.23 or later.
+-   If you are running on `macOS`, make sure Docker is running
```
Comment (afrittoli) [inline docs/install.md:25]: Why only when running `macOS`? Why Docker?
This is a list of prerequisites for installing Tekton on Kubernetes - the first in the list is a Kubernetes cluster - the procedure is not really specific to where that kubernetes cluster is running.

**Comment by afrittoli** [comment]
Thank you @pavanstarmanwar for your contribution!

The old installation guide was referring to `kind` specifically, but not the current one.
In the current guide, the installation instructions start after a couple of paragraphs about setting up a local cluster (https://tekton.dev/docs/installation/local-installation/) which refer to the `kind` and `minikube` documentation directly.

The starting point for this document is a running k8s cluster, so I don't think that docker and troubleshooting about `kind` do not belong in it. 

FYI @geriom @pritidesai 

I would propose to revert this PR.

**Comment by geriom** [comment]
> The starting point for this document is a running k8s cluster, so I don't think that docker and troubleshooting about `kind` do not belong in it.

> I would propose to revert this PR.

I agree with you. This information is specifically about `kind` and should be covered by the Kind documentation. 

I'm reverting the changes in #6172 which also adds a bunch of missing links.

**Pull request merged**